packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx

```diff
@@ -92,6 +76,7 @@ const StyledFormFeedbackTooltip = styled.div<{
 
     ${media.largeMobile(css`
       width: auto;
+      border-radius: ${theme.orbit.borderRadiusLarge};
```

Comment:
This is actually not correct. It should be Large on default and on desktop (and bigger) should be Normal.

I also checked with Will and it should also be applied to the actual input, and not only the tooltip.